core/contracts/financial-templates/implementation/SyntheticToken.sol

```diff
@@ -4,76 +4,73 @@ import "../../common/implementation/ExpandedERC20.sol";
 
 
 /**
- * @notice A burnable and mintable ERC20. The contract deployer will initially
- * be the only minter and burner as well as the owner who is capable of adding new roles.
- * The contract deployer will also be the only initial minter of the contract.
+ * @title Burnable and mintable ERC20.
+ * @dev The contract deployer will initially be the only minter and burner as well as the owner  who
+ * is capable of adding new roles. The contract deployer will also be the only initial minter of the contract.
  */
+
 contract SyntheticToken is ExpandedERC20, ERC20Detailed {
+    /**
+     * @notice Constructs the SyntheticToken
+     * @param tokenName used to describe the new token.
+     * @param tokenSymbol short ticker abbreviation of the name. Ideally < 5 chars.
+     * @param tokenDecimals used to delimitate the new token.
+     */
     constructor(string memory tokenName, string memory tokenSymbol, uint8 tokenDecimals)
         public
         ERC20Detailed(tokenName, tokenSymbol, tokenDecimals)
     {}
 
     /**
-     * @dev Add minter role to account.
-     *
-     * Requirements:
-     *
-     * - the caller must have the Owner role.
+     * @notice Add minter role to account.
+     * @dev the caller must have the Owner role.
```

Comment:
If `Owner` role is capitalized, then should also make `Minter` role capitalized.

Reply:
good point. fixed